Document the launch button functionality (#2046)
jupyter-book/mystmd: docs/website-launch-buttons.md

---
title: Launch Jupyter sessions
---

> If your source file is a Jupyter Notebook (`.ipynb`), you can add a button to your MyST website that launches a live Jupyter server in the cloud, and opens that content so they can edit and interact with it.[^thebe]

Reviewer: ```suggestion
You can add a button to your MyST website that launches a Jupyter environment in the cloud and opens the current file in the editor. If the file is a Jupyter Notebook (`.ipynb` file) or a MyST document with code-cells[^lab-myst], users can use the Jupyter environment to run the code cells and inline expressions.[^thebe]
```

Reviewer: The important points I'm trying to make here are:

1. You don't actually need the current doc to be executable (my mistake over Slack) for the button to appear on a page.
2. If the environment understands MyST documents with kernelspecs, they too can be executed.

> [^thebe]: This is a fully-interactive Jupyter environment. If you'd instead like like to provide interactivity to the same page, see [](./integrating-jupyter.md).

Reviewer: ```suggestion
[^thebe]: This is a fully-interactive Jupyter environment. If you'd instead like like to provide interactivity to the same page, see [](./integrating-jupyter.md).
[^lab-myst]: Interacting with MyST Markdown documents is only supported in JupyterLab with the `jupyterlab-myst` extension installed.
```

---
title: Launch Jupyter sessions
---

You can add a button to your MyST website that launches a Jupyter environment in the cloud and opens the current file in the editor. If the file is a Jupyter Notebook (`.ipynb` file) or a MyST document with code-cells[^lab-myst], users can use the Jupyter environment to run the code cells and inline expressions.[^thebe]

[^thebe]: This is a fully-interactive Jupyter environment. If you'd instead like like to provide interactivity to the same page, see [](./integrating-jupyter.md).

> [^lab-myst]: Interacting with MyST Markdown documents is only supported in JupyterLab with the `jupyterlab-myst` extension installed.

Author: ```suggestion
[^lab-myst]: Interacting with MyST Markdown documents is only supported in JupyterLab with [the `jupyterlab-myst` extension](https://jupyter-book.github.io/jupyterlab-myst/) installed.
```
